client/gutenberg/extensions/contact-form/components/jetpack-contact-form.jsx

```diff
@@ -262,9 +264,15 @@ class JetpackContactForm extends Component {
 						/>
 					) }
 					{ has_form_settings_set && (
-						<div className="button button-primary button-default jetpack-submit-button">
-							{ submit_button_text ? submit_button_text : __( 'Submit' ) }
-						</div>
+						<PlainText
+							value={ submit_button_text }
+							className="button button-primary button-default jetpack-submit-button"
+							onChange={ value => {
+								this.setState( { inFocus: null } );
```

Comment:
We don't seem to need `inFocus` in this component. Could you elaborate on why we need this here?